(sakuli/sakuli-docker#19) added doc for git clone in container (#59)
sakuli/docs: content/enterprise_features/e2e.md

> When working with added files and folders inside a container, one has to ensure correct file permissions for added files.

#### 4.1.3 Clone a git repository

The Sakuli container provides a mechanism to clone a git repository at the container start and executing the sakuli project
within it:
{{<highlight bash>}}

> docker run -e GIT_URL=<REPOSITORY URL> -e GIT_CONTEXT_DIR=<SAKULI TESTSUITE PATH IN CLONED REPO> -e SAKULI_LICENSE_KEY=<YOUR SAKULI LICENSE KEY> taconsol/sakuli:2.3.0

Reviewer: ```suggestion
docker run -e GIT_URL=<REPOSITORY URL> [-e GIT_CONTEXT_DIR=<RELATIVE PATH TO TESTSUITE >] -e SAKULI_LICENSE_KEY=<YOUR SAKULI LICENSE KEY> taconsol/sakuli:2.3.0
```

Missing optional braces on `GIT_CONTEXT_DIR`

| VNC_VIEW_ONLY           | false                | Enable/Disable view-only mode                       |
| NPM_TOKEN               |                      | NPM token to access npmjs.com registry              |
| SAKULI_TRUSTED_CERT_DIR |                      | Directory containing custom certificates for import |
| GIT_URL                 |                      | URL of git repository                               |

> | GIT_DIR                 | /headless/git-repository| Directory where the git repository is cloned     |

Reviewer: Why would we need this configurable at all? Convention > Configuration. IMHO, it would be enough to clone the git repo somewhere we expect it during container runtime. Omitting `GIT_DIR` would also shorten the docker command and remove complexity from the customers resposibilities.

| VNC_VIEW_ONLY           | false                | Enable/Disable view-only mode                       |
| NPM_TOKEN               |                      | NPM token to access npmjs.com registry              |
| SAKULI_TRUSTED_CERT_DIR |                      | Directory containing custom certificates for import |
| GIT_URL                 |                      | URL of git repository                               |

> | GIT_CONTEXT_DIR         |                      | Path to Sakuli testsuite within the git repository  |

Reviewer: ```suggestion
| GIT_CONTEXT_DIR         | .                    | Path to Sakuli testsuite within the git repository  |
```

I'm missing the default value here.
